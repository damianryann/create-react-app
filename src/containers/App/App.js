import React from 'react';

import './App.scss';
import logo from '../../assets/images/logo.svg';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <div className="container">
          <div className="row">
            <div className="col">
              <img src={logo} className="App-logo" alt="logo" />
              <p>
                Edit <code>src/containers/App/App.js</code> and save to reload.
              </p>
              <a
                className="App-link"
                href="https://reactjs.org"
                target="_blank"
                rel="noopener noreferrer"
              >
                Learn React
              </a>
            </div>
          </div>
        </div>
      </header>
    </div>
  );
}

export default App;
