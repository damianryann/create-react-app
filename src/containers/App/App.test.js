import React from 'react';
import { mount } from 'enzyme';

import App from './App';

it('mounts without crashing', () => {
  const component = mount(<App />);

  expect(component).toMatchSnapshot();
});
