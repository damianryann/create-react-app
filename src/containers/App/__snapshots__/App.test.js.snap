// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mounts without crashing 1`] = `
<App>
  <div
    className="App"
  >
    <header
      className="App-header"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col"
          >
            <img
              alt="logo"
              className="App-logo"
              src="logo.svg"
            />
            <p>
              Edit 
              <code>
                src/containers/App/App.js
              </code>
               and save to reload.
            </p>
            <a
              className="App-link"
              href="https://reactjs.org"
              rel="noopener noreferrer"
              target="_blank"
            >
              Learn React
            </a>
          </div>
        </div>
      </div>
    </header>
  </div>
</App>
`;
